README.md

```diff
@@ -47,9 +47,26 @@ See [CHANGELOG](https://github.com/metosin/compojure-api/blob/master/CHANGELOG.m
     (ok {:message (str "Hello, " name)})))
 ```
 
+### Validating an API
```

Comment:
The old README had all the details in it and it was really hard to read. I think validation is a details that should be in the wiki, not in README. Could you move this part to wiki?